### PR TITLE
feat: Add import reader for Parquet

### PR DESCRIPTION
**Pull request by bigsheeper**
This PR implements a Parquet reader for import.

issue: https://github.com/milvus-io/milvus/issues/28521

### CODE REVIEW

File: internal/util/importutilv2/parquet/column_reader.go [hunk 1]
```diff
@@ -0,0 +1,561 @@
...
+func isRegularVector(offsets []int32, dim int, isBinary bool) bool {
+	if len(offsets) < 1 {
+		return false
+	}
+	if isBinary {
+		dim = dim / 8
+	}
+	start := offsets[0]
+	for i := 1; i < len(offsets); i++ {
+		if offsets[i]-start != int32(dim) {
```
Comment (xiaocai2333) [inline internal/util/importutilv2/parquet/column_reader.go:433]: offsets[i] - offsets[i-1] != dim ?

**Review by xiaocai2333 — COMMENTED** [review]
/lgtm

**Comment by samhuang-z** [comment]
rerun ut

**Comment by bigsheeper** [comment]
rerun ut

**Comment by bigsheeper** [comment]
rerun ut

**Comment by bigsheeper** [comment]
rerun ut

**Pull request merged**